epictrack-api/src/api/schemas/request/work_request.py

```diff
@@ -26,13 +26,13 @@ class WorkBodyParameterSchema(RequestBodyParameterSchema):
         required=True,
     )
 
-    short_description = fields.Str(
-        metadata={"description": "Short description of work"},
+    report_description = fields.Str(
+        metadata={"description": "Report description of work"},
         validate=validate.Length(max=500),
         required=True,
     )
-    long_description = fields.Str(
-        metadata={"description": "Long description of work"},
+    epic_description = fields.Str(
```

Comment:
Any particular reason for this name change. Cz the long description would serve the purpose. In the front end we keep changing the label, but it may not be possible to change the backend column name all the time.

Reply:
Only reason for changing the backend column name is so it matched what we have on the front end. Otherwise we just have to remember that report description (frontend) is the short description (backend) and that epic description (frontend) is long description (backend).

Reply:
But if you want I can revert all the backend changes